docs/src/api/class-android.md

```diff
@@ -202,6 +202,12 @@ Prevents automatic playwright driver installation on attach. Assumes that the dr
 Optional device serial number to launch the browser on. If not specified, it will
 throw if multiple devices are connected.
 
+### option: Android.launchServer.host
+* since: v1.5
+- `host` <[string]>
+
+Host to use for the web socket. Defaults to `unspecified`.
```

Comment:
Your change looks good. Please remove the mentions of 'unspecified', we imply that for the options, they are optional.